### PR TITLE
Create CRUD for Module without edit

### PR DESCRIPTION
**Pull request by MikolaLvivUA**

### CODE REVIEW

File: src/routes/module/module.router.ts [hunk 1]
```diff
@@ -6,4 +6,11 @@ const router = Router();
 
 router.post('/', moduleController.createModule);
 
+router.get('/', moduleController.getAllModules);
+router.get('/:module_id', moduleController.getModuleById);
+
+router.patch('/:module_id', moduleController.editModule);
```
Comment (VictorKmin) [inline src/routes/module/module.router.ts]: isModulePresentMiddleware

File: src/services/module/module.service.ts [hunk 2]
```diff
@@ -1,4 +1,7 @@
-import { Module } from '../../database';
+import { model } from 'mongoose';
+
+import { DatabaseTablesEnum } from '../../constants/enums';
```
Comment (VictorKmin) [inline src/services/module/module.service.ts]: import { DatabaseTablesEnum } from '../../constants';

File: src/routes/module/module.router.ts (hunk 1 above)
[the remark this answers is not in the document]
Comment (MikolaLvivUA) [reply src/routes/module/module.router.ts]: fixed

Comment (MikolaLvivUA) [reply src/routes/module/module.router.ts]: fixed

File: src/services/module/module.service.ts (hunk 2 above)
Comment (MikolaLvivUA) [reply src/services/module/module.service.ts]: fixed

**Pull request merged**